### PR TITLE
Extend expiring switches

### PR DESCRIPTION
**Pull request by DanielCliftonGuardian**
## What is the value of this and can you measure success?
Enables deployment

### CODE REVIEW

File: common/app/conf/switches/JournalismSwitches.scala [hunk 1]
```diff
@@ -81,7 +81,7 @@ trait JournalismSwitches {
     description = "Show the Euro 2024 interactive atom header on football pages",
     owners = Seq(Owner.withEmail("dotcom.platform@theguardian.com")),
     safeState = Off,
-    sellByDate = LocalDate.of(2024, 7, 31),
+    sellByDate = LocalDate.of(2024, 8, 30),
```
Comment (JamieB-gu) [inline common/app/conf/switches/JournalismSwitches.scala:84]: I think we're removing this one?

File: common/app/conf/switches/ABTestSwitches.scala [hunk 2]
```diff
@@ -31,7 +31,7 @@ trait ABTestSwitches {
     "Test MPU when there is no epic at the end of Article on the page.",
     owners = Seq(Owner.withEmail("commercial.dev@theguardian.com")),
     safeState = Off,
-    sellByDate = Some(LocalDate.of(2024, 7, 31)),
+    sellByDate = Some(LocalDate.of(2024, 8, 30)),
```
Comment (JamieB-gu) [inline common/app/conf/switches/ABTestSwitches.scala:34]: I think this is handled in #27367

File: common/app/conf/switches/ABTestSwitches.scala [hunk 3]
```diff
@@ -41,7 +41,7 @@ trait ABTestSwitches {
     "Show new ad block ask component in ad slots when we detect ad blocker usage",
     owners = Seq(Owner.withEmail("commercial.dev@theguardian.com")),
     safeState = Off,
-    sellByDate = Some(LocalDate.of(2024, 7, 31)),
+    sellByDate = Some(LocalDate.of(2024, 8, 30)),
```
Comment (JamieB-gu) [inline common/app/conf/switches/ABTestSwitches.scala:44]: I think this is handled in #27367

File: common/app/conf/switches/ABTestSwitches.scala [hunk 4]
```diff
@@ -51,7 +51,7 @@ trait ABTestSwitches {
     "A sticky reader revenue ask on the left column of a liveblog",
     owners = Seq(Owner.withEmail("growth@theguardian.com")),
     safeState = Off,
-    sellByDate = Some(LocalDate.of(2024, 7, 31)),
+    sellByDate = Some(LocalDate.of(2024, 8, 30)),
```
Comment (JamieB-gu) [inline common/app/conf/switches/ABTestSwitches.scala:54]: I think @sookburt is planning to handle this one?

**Review by domlander — APPROVED** [review]

**Pull request merged**